src/main/java/org/overture/ego/model/enums/UserRole.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017. The Ontario Institute for Cancer Research. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.overture.ego.model.enums;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public enum UserRole {
+  USER("USER"),
+  ADMIN("ADMIN");
+
+  @NonNull private final String value;
+
+  @Override
+  public String toString() {
+    return value;
+  }
```

Comment:
I like a new line between closing brace of last method and closing brace of class/enum